cmd/allocator/main.go

```diff
@@ -202,7 +244,8 @@ func getCACertPool(path string) (*x509.CertPool, error) {
 				return nil, fmt.Errorf("ca cert is not readable or missing: %s", err.Error())
 			}
 			if !caCertPool.AppendCertsFromPEM(caCert) {
-				return nil, fmt.Errorf("client cert %s cannot be installed", certFile)
+				logger.Errorf("client cert %s cannot be installed", certFile)
```

Comment:
why isn't this an error any longer?

Reply:
Because if a customer mistakenly add a bad client certificate to the authorized list, their service should handle it gracefully instead of crashing.

Reply:
Then should the return above (line 249) also log but not return an error?

Reply:
Good catch. Thanks!